config/game_options.txt

```diff
@@ -77,8 +77,8 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
 ## Set to 0 to disable that mode.
 
 PROBABILITY TRAITOR 5
-PROBABILITY TRAITORBRO 2
-PROBABILITY TRAITORCHAN 4
+PROBABILITY TRAITORBRO 4
+PROBABILITY TRAITORCHAN 2
 PROBABILITY INTERNAL_AFFAIRS 3
 PROBABILITY NUCLEAR 2
 PROBABILITY REVOLUTION 2
```

Comment:
Raise cult up two notches. It should be more common.

Reply:
Knock wizard down two notches. It shouldn't be more common than changeling.

Reply:
redundant